sms-checker/test_message.py

```diff
@@ -0,0 +1,63 @@
+'''Contains tests for message.py'''
```

Comment:
A wholistic test suit for ensuring the functions that send emails and text messages are called in an expected manner, and that the format of messages that reach subscribers is correct.